cvmfs/publish/repository.cc

```diff
@@ -576,18 +576,18 @@ void Publisher::InitSpoolArea() {
   CreateDirectoryAsOwner(settings_.transaction().spool_area().cache_dir(),
                          kPrivateDirMode);
   CreateDirectoryAsOwner(settings_.transaction().spool_area().scratch_dir(),
-                         kPrivateDirMode);
+                         kDefaultDirMode);
   CreateDirectoryAsOwner(settings_.transaction().spool_area().ovl_work_dir(),
-                         kPrivateDirMode);
+                         kDefaultDirMode);
```

Comment:
I think the overlay work directory should be kept with private permissions.

Reply:
Done!